src/main/java/io/appform/functionmetrics/FunctionMetricsManager.java

```diff
@@ -30,16 +31,12 @@
  * Global metrics manager that needs to be initialized at start
  */
 public class FunctionMetricsManager {
-    private static final Logger log = LoggerFactory.getLogger(FunctionMetricsManager.class.getSimpleName());
+    private static final Logger log = LoggerFactory.getLogger(FunctionMetricsManager.class.getName());
```

Comment:
Why this change ?

Reply:
To maintain the same standard of logging configuration across all the libraries. Simple name returns just "FunctionMetricsManager" whereas it should be "io.appform.functionmetrics.FunctionMetricsManager". Using the fully qualified package name would help us apply logging configuration changes in hierarchical fashion.